README.md

```diff
@@ -1,4 +1,4 @@
 ### Portfolio
 
-This is my Portfolio.
+This is my Portfolio written in html & cssd.
```

Comment:
@rpatil1998 can you make this to `css` instead of `cssd`.

Reply:
Lol.

Reply:
Lol